catalog/security/catalog-security-plugin/src/test/java/ddf/catalog/security/plugin/SecurityPluginTest.java

```diff
@@ -68,6 +71,36 @@ public void testNominalCaseCreate() throws Exception {
                         .getValue(), equalTo(TEST_USER)));
     }
 
+    private Subject setupMockSubject() {
```

Comment:
This was actually a last minute change that I put in when I realized that a unit test was failing! I will move it down.